src/celengine/render.cpp

```diff
@@ -247,11 +236,10 @@ Renderer::Renderer() :
     windowWidth(0),
     windowHeight(0),
     fov(FOV),
-    cosViewConeAngle(computeCosViewConeAngle(fov, 1, 1)),
+    cosViewConeAngle(std::sqrt(3.0)),
```

Comment:
why 3.0?

Reply:
this should be 1 / sqrt(3), since the old computeCosViewConeAngle(fov, 1, 1) with fov = 45 deg would return this value.

Reply:
I'm asking why do you use a magic constant? please add a `const` or `constexpr` variable with comment why its value is such.

Reply:
i'll set to 0 since the initial value does not realy matter, it gets calculated before usage.